code/modules/mob/transform_procs.dm

```diff
@@ -281,6 +281,31 @@
 	qdel(src)
 	return new_corgi
 
+/**
+ * Turns the source atom into a crab crab, the peak of evolutionary design.
+ * WARNING! This is IRREVERSIBLE!
+ * It adds the NO_TRANSFORM trait and should thus be reserved for admin shenanigans or effective round-enders for the most part.
+ */
+/mob/living/carbon/human/proc/crabize()
+	if(HAS_TRAIT(src, TRAIT_NO_TRANSFORM))
```

Comment:
Just to clarify, you know what `TRAIT_NO_TRANSFORM` does right? Because it's not "block transforming into another mob".

Reply:
hey. Hey. i copied this whlesale from corgize(). its not my fault.

Reply:
you’re responsible for what you copy 🙂